### PR TITLE
Big Red Last Stand Update

### PR DESCRIPTION
**Pull request by PalJohnson**
## About The Pull Request

- This update is seeking to shake up the survivor gameplay by providing unique scenarios for survivors. This update adds several survivor holdouts (neutral & hostile) for survivors. In addition I added a minor bugfix for synth survivor,

## Why It's Good For The Game

- One of the most underrated aspects of survivors is the ability to be able to do a last stand. The current meta shift revolves around LZ holds or roaming if that's not possible. This update should encourage survivors not to leave their place of work and instead fight and in most cases die for their company if they feel inclined to as opposed to roaming or cheesing out LZ holds.

## Non-Goals

- Giving survivors additional gear. Given that their gear is random and spread across the maps survivors will now have a focal point to restock on valued items in a localized area and runoff. Although survivors are expected to stay and fight to the last colonist, nothing stops them from looting the place. In the end of the day its player agency and I don't think it would be good to force them to do something they don't want to do like die in the bar, library or marshalls. Given that most the CLF insert have 25% spawn rate, as well as the dnd, insert this shouldn't be a problem. Given the frequency of the bar holdout, I have purposely given them fewer resources.
- 
- Xenomorphs have a harder time killing survivors. Although killing or capturing survivors is not a xenomorph goal as they can ignore them entirely. Having a holdout at places like bar or even libraries for that matter could prove to be a nuisance due to premade defenses and gear being available. As stated previously nothing guarantees the survivors staying there in the first place so they could be unmanned and easily melted. Don't assault a survivor holdout alone.

## Changelog

_**- Maps**_

This update as mentioned adds 3 new survivor holdouts for survivors that will randomly spawn throughout the map. Currently, there are only 3 being added but more will be added in the future. The colony didn't get raptured it was a bloody mess with people crying and fighting a hell hole. The MR has been balanced using the CLF crashed ship insert on LV-624 as an example as to how much medical, guns and other items will be spawned in for survivors. These are prepared positions so they should have ok gear and protection if we actually want to encourage people to hold it.

_**CLF Marshalls Take Over**_ **(25% Chance)**
image

- As stated previously the CLF have invaded the colony and have taken over the last bastion of the authority of the colony that is the colonial marshall's offices and armory. After the successful takeover, the CLF decided to make the ruins of the office into their own area of operation for future CLF activities in the colony. much like the crashed ship insert, they have been equipped with identical gear to their LV-624 counterparts with minor exceptions. The exception is materials to repair and potentially expand their area of influence. The barricades for the most part are worn out and need some repairs if they expect to live long enough.

_**Dungeons and Dragons Last Stand**_ **(25% Chance)**
image
- This nightmare insert was inspired by the event held by jarek where survivors attempted to have a sleepover in the LV nexus dormitories only to be slaughtered. This insert mirrors that in that colonists are hosting a dnd session in the local library. Given that the civilian survivor is rather lackluster I have gone ahead and supplied them with cosplay material so that they can arm themselves and carry out their dnd session in style. Much of it is fluff in many regards with the exception of the illegally smuggled merc gear. The balance of the insert is similar in the crashed ship insert but the barricades are repaired and the shutters closed.

_**The Bar Last Stand**_ **_(40% Chance)_**
image
- This insert reflects the valiant efforts of the colonist to have a last stand within the colony bar. The bar (according to the colonist handbook) was designated as an emergency rally point for all personnel and staff in the event of an unforeseen event. They come equipped with standard wooden barricades with building material should they want to replace them. In addition, they have ammo, medical supplies, and liquid courage to help them in their endeavor.

_**Minor Updates**_

- Synth survivor now has colonist radio. Previously in the synth survivor update, I accidentally gave them the wrong radio. This prevented them from communicating with survivors as they had a unique radio. The auto lathes produce marine radios so they couldn't find an adequate communication device.

- Colonist survivor now has the colonist jumpsuit. They had PJs for some reason. Odd

:cl:
add: Added three new survivor holdout nightmare inserts for big red
add: Added clf survivor to big red with their own unique story to go with it
expansion: Expanded existing mechanics or gameplay
fix: fixed synth survivors not having colonists' radio
fix: fixed civilian survivor not having actual colonist clothing
/:cl:

### CODE REVIEW

**Comment by Agameofscones** [comment]
```
loading map_files/BigRed/BigRed.dmm
maps/map_files/BigRed/BigRed.dmm:19109:error: bad type or variable: dnd
maps/map_files/BigRed/BigRed.dmm:19109:error: expected )
```
You have bad instances. Fix em.

**Comment by PalJohnson** [comment]
Sure thing. Im not sure how I got it though.

**Comment by PalJohnson** [comment]
> ```
> loading map_files/BigRed/BigRed.dmm
> maps/map_files/BigRed/BigRed.dmm:19109:error: bad type or variable: dnd
> maps/map_files/BigRed/BigRed.dmm:19109:error: expected )
> ```
> 
> You have bad instances. Fix em.

resolved it. It was missing quotation marks for some reason.

**Comment by c4xmaniac** [comment]
I'm pretty sure a lot of us surv players have talked about having the meta shifted to where we all spawn together, its nice to see this being implemented. I think this will also greatly help newer players because when its your first time as survivor you'll be with people who know what they're doing (hopefully). This has a side benefit of also deincentivizing locker hiders from afking in lockers for 30-40min since we all start together. 

I don't think that CLF spawns, or hostile survs in general, are an issue if they are telegraphed. This is a good example, like the LV CLF survs, where upon landing and seeing the marshals scenario you will immediately know that there are, or were, hostile survs around. I think it gives good RP opportunities by either the surv synth negotiating with the xenos and the CLF to make an enemy of my enemy kind of pact, or vice versa with the marines. If the survs decide to actually be fully hostile, its going to be pretty difficult to survive not only the xenos but also the marines gunning for them. I'm not going to pretend that I'm going to be super nice and not grief marines as CLF, because I am, but I think it offers some cool scenarios where you're about to lose garage and the people coming to theoretically save you are people who hate you and will take you prisoner/kill you outright. 

The main problems I foresee with the holds is that, for instance, while the bar is genuinely a cinematic hold and would fit right into the Aliens movie, survivor players are in TDM mode from start to finish. Due to the poor location of the bar, I foresee it being looted then abandoned for a superior location such as marshals, garage, or engineering. Equally similar of an issue is the dnd hold. Its a funny hold, and I really enjoyed the slumber party hold we did on LV, or the pod/dnd holds on Science Annex, but most people aren't You, Greta, Brad, Amber, or myself and will fuck off to a more superior position instead of playing dnd while xenos are knocking down our door.

It should also be noted that some of these positions would be difficult for lowpop xenos to overrun. If you have one or two good survivors, its insanely easy to hold low pop because usually only one-two xenos bother harassing you until you get bored and go hunting for them. Though, I don't think the game should be balanced around low pop, and like I said, even without premade fortifications its easy to hold lowpop with a standard marshals or engineering hold.

To summarize my wall of text, I think its good to see survivor being developed towards inclining players towards RP and surviving rather than being a hit squad. I hope that you also consider adding back in the flavor text of what you were doing with whom before the outbreak and more randomized civilian jobs to help hammer it home that we're survivors of a colony and not seal team  6.

**Comment by PalJohnson** [comment]
> 

I plan to add back the civilian roll much in the same way that the synth survivor role was. It will come in a later MR but after I push out the rest of the Last stand updates there is one for every map (other than soro). 

As for the low pop I'm really not that concerned about it. I balanced the prepared areas around our average server pop of around 120 players. Even on low pop survivors won't be able to benefit from the variety of defenses available and can be easily stretched thin.

The bar hold was done so that it would invite a small portion of danger admittedly. Its a focal point of conflict so I thought it was best to place it there as opposed to an area like marshalls which can be some distance away from combat. The DND location was done due to the library being where its at. I would consider making the dnd hold (given how goofy and rare it is) a lot more reinforced so that the xenos could only break through the windows much like the LV secure storage. Perhapse a doomsday bunker scenario.

**Comment by stanalbatross** [comment]
as these are spawned via nightmare, I would advise against adding them in right now until we fix the maploader issues that make nightmare inserts "duplicate", lest we have even more issues on big red.

**Comment by PalJohnson** [comment]
> as these are spawned via nightmare, I would advise against adding them in right now until we fix the map loader issues that make nightmare inserts "duplicate", lest we have even more issues on big red.

They use sprinkles But I assume it uses the same process

**Comment by PalJohnson** [comment]
said to reopen by stan
